src/components/data-testid/decision-tree-modal.cy.tsx

```diff
@@ -0,0 +1,17 @@
+export const DecisionTreeModal = {
```

Comment:
I'd add `DataCy` postfix to this constant, because it's not obvious that you import selectors and not modal component. Basically should be the same as for other similar constants.